inode.hpp

```diff
@@ -82,6 +82,12 @@ struct inode {
     #if defined(PLATFORM_WINDOWS)
     //on Windows, the last status change time (ctime) holds the file creation time instead
     case time::create: return data.st_ctime;
+    #elif defined(__OpenBSD__)
+    // OpenBSD is a special case that must be handled separately from other BSDs
+    case time::create: return min((uint)data.__st_birthtime, (uint)data.st_mtime);
+    #elif defined (__DragonFly__)
+    // DragonFly BSD does not support file creation time, use modified time instead
+    case time::create: return data.st_mtim.tv_sec;
```

Comment:
Why is this `st_mtim.tv_sec` and not `st.mtime`?

Reply:
I did have some rationale for choosing this over st.mtime at the time I wrote it, but at this point in time I can't remember what that was. st.mtime does indeed work fine.

Here is where it is defined in the DragonFly BSD sources:
https://github.com/DragonFlyBSD/DragonFlyBSD/blob/master/sys/sys/stat.h#L125

I'm happy to change this to st.mtime if this is preferable. Please let me know and it shall be done.

Reply:
The other platform implementations fall back to `st_mtime`, so for consistency let's do that here, too.